### PR TITLE
Pull Requests

### PR DESCRIPTION
**Pull request by pnw**
I was asked to write up a convention for commenting on pull requests and adding an importance rating.

### CODE REVIEW

**Comment by pnw** [comment]
@anotheredward @boredsteve @markstuart @matt-in-a-hat @Resonance1584 @digitalsadhu @jooleearr @msandrew @JonathanPrince @JonathanPrince @ebuckley @ewen @tfentonz 

Did I miss anyone?

**Comment by boredsteve** [comment]
missed @jonforest and stu I think!

File: pull-requests.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+When reviewing another person's code, it is not beneficial to censor yourself by omitting comments that you think might be trivial or inconsequential, since the seemingly casual observations sometimes turn into meaningful conversations or decisions. In order to facilitate "See Something, Say Something" I've adopted a rating system to differentiate important comments from casual observations and everything in between.
+
+## Rating the importance of your comment
+
+This rating system is used to indicate how strongly you feel about a particular comment. At MediaSuite, we do not give the
```
Comment (matt-in-a-hat) [inline pull-requests.md]: 9/10 What don't we give? A flying f***?

Comment (pnw) [inline pull-requests.md]: I think I part of the sentence

**Comment by ebuckley** [comment]
@metapraxis 

Welcome to mediasuite tech-stack documentation!

**Comment by ebuckley** [comment]
Nice work, I like the reappropriation of propaganda up in here.

The codification of seriousness of comments is important, but at the moment, the document is missing the reinforcement aspect code review. If you see something awesome, mention it!  This can be a pretty good way to show you are actually doing a full code review in cases where the only output would be a `LGTM :shipit:` comment.

**Comment by pnw** [comment]
Good shout @ebuckley I added a section for positive feedback.

**Comment by anotheredward** [comment]
If we're making a pull request document, I did a talk on Code Review: 
https://www.youtube.com/watch?v=p83ohn3E3Dg
https://docs.google.com/presentation/d/1H2kf61kFrlqZtXSbWWHZYTK2F4xJ3by3GDhDKdoSUtw/edit#slide=id.p17
And wrote a checklist, which nobody uses: 
https://docs.google.com/document/d/1SLtnmd9moWfudtPvyNzwvKFgBRiiGI5NbpRz_zvksWM/edit

File: pull-requests.md [hunk 2]
```diff
@@ -34,5 +39,13 @@ A good default format for a PR comment:
...
+<number>/10 <optional: "IMO" for the real hesitant types> <action to take or reason why I think this is important or both>
 ```
+
+## Positive Feedback
+
+tl;dr Always use 0/10 for positive feedback that requires no action.
+
+Despite what government scare tactics may have you believe, good things are worth Saying Something about too. When giving positive feedback, e.g. "this is cool", "I like what you did here", "wow these examples are super meaningful and useful", it's best to give your comment a 0/10 rating so the author knows that you are just giving feedback and not suggesting actions. 
+
+If the zero-ness of your actionless good-vibes comment bothers you, you can always rate your comment in [Verlan](https://en.wikipedia.org/wiki/Verlan) style, or 10/0. I would not recommend that approach considering 10/0 happens to be infinity which is incomprehensible to you mortals.
```
Comment (ebuckley) [inline pull-requests.md]: 10/0 #hedidthemath
screen shot 2016-08-25 at 10 01 30 am

**Comment by ebuckley** [comment]
LGTM :shipit: 

lets merge

**Pull request merged**